proposals/lip-0051.md

```diff
@@ -2447,7 +2448,9 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
   * The `supplySubstore` array must be in ascending order of `tokenID`.
```

Comment:
small typo on line 2447, just caught it: "across all element" --> "across all elements"